File: submissions/sm_007_brijesh/week_3/day_2/assignment/taxCalculator.js
function tax_cal(income, saving) {
    var income = income;
    var saving = saving;
    var tax = 0;
    var rebate = 0;


    if (income < 250000) {
        console.log('No Tax')
    }

    else {
        if (income > 250000 && income < 500000) {
            rebate = 0.5 * saving;
            if (rebate > 50000) {
                rebate = 50000;
            }
            income = income - rebate;
            console.log(rebate)
            income = income - 250000;
            tax = 0.1 * income;
            /* tax = ((income - rebate)-250000) * 0.1 */
        }
        else if (income > 500000 && income <= 1000000) {
            rebate = 0.3 * saving;
            if (rebate > 50000) {
                rebate = 50000;
            }
           /* income = income - rebate;
            console.log(rebate)
            income = income - 250000;
            tax = 0.2 * income; */
            console.log(rebate);
           tax = ((income - rebate) - 500000) * 0.2;

        }
        else if (income > 1000000){
        rebate = 0.1 * saving;
        if(rebate > 50000){
            rebate = 50000;
        }
        tax = ((income - rebate) - 1000000) * 0.3;

        }


}
console.log(tax)
}
tax_cal(1200000, 180000)

File: submissions/sm_015_lalit/week_13/day_5/session_1/tax_calculator.py
import math
inputIncome=input("Provide input for income here : ")
inputRebate=input("Provide input for rebate now : ")
inputIncome=int(inputIncome)
inputRebate=int(inputRebate)

def taxCal(incomex,saving):
    saving=int(saving)
    incomex=int(incomex)
    tax=0
    if 250000 >= incomex:
        tax=0
    elif 500000 >= incomex:
        rebate=saving*0.5
        if rebate > 50000:
            rebate=50000
        incomex=incomex-(rebate)
        income1=incomex-250000
        tax=income1*0.1
    elif 1000000 >= incomex and incomex > 500000:
        rebate=saving*0.3
        if rebate > 50000:
            rebate=50000
        incomex=incomex-(rebate)
        income1=incomex-250000
        tax=250000*0.1
        income2=income1-250000
        tax+=income2*0.2
    elif incomex>1000000:
        rebate=saving*0.1
        if rebate > 50000:
            rebate=50000
        incomex=incomex-(rebate)
        income1=incomex-250000
        tax=250000*0.1
        income2=income1-250000
        tax+=500000*0.2
        income3=income2-500000
        tax+=income3*0.3

    return math.floor(tax)

print("Tax of Rs",str(taxCal(inputIncome,inputRebate)),"is calculated on the basis of your input")        


    
    